index.html

```diff
@@ -0,0 +1,219 @@
+                            </div>
+                              <h1 class="work-title">Multi-Post<br>
+                                Stories</h1>
+                              <div class="work-role">
+                                <h3 class="work-role-list role-job ">CANOPY</h3>
+                                <div class="work-role-list  role-dot">&#8901;</div>
+                                <div class="work-role-list  role-name">Back end dev</div>
+                                <div class="work-role-list  role-dot">&#8901;</div>
+                                <div class="work-role-list  role-year">2023</div>
+                              </div>
+                              <p class="project-description">
+                                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                </p>
+                                <ul class="my-skill">
+                                    <li><button title="html">html</button></li>
+                                    <li><button title="css">css</button></li>
+                                    <li><button title="js">Javascript</button></li>
+                                </ul>
+                                <span class="p-more-information">
+                                  <a class="moresee-link">See project</a>
+                                  </span>
+                            </span>
+          </div>
+</section> 
+<section class="aboutmyself-section">
+    <div id="aboutmyself-container">
+      <span class="aboutmyself-content">
+<h1 class="aboutmyself-title intro-name">About<br>
+    Myself</h1>
+    <p class="aboutmyself-history intro-position">
+        I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! If you like what you see 
+        and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <p><br><a class="intro-connect">LET’S CONNECT</a><br></p>
+    <ul class="view-social-midea">
+        <li class="social-icon">
+            <a><img src="./assets/images/social-icon/twitter-icon.png" alt="twitter"></a>
+        </li>
+        <li class="social-icon">
+            <a><img src="./assets/images/social-icon/linkedin icon.png" alt="linkedin"></a>
+        </li>
+        <li class="social-icon">
+            <a><img src="./assets/images/social-icon/m-icon.png" alt="m"></a>
+        </li>
+        <li class="social-icon">
+            <a><img src="./assets/images/social-icon/github-icon.png" alt="github"></a>
+        </li>
+        <li class="social-icon">
+            <a><img src="./assets/images/social-icon/clip-icon.png" alt="fingers"></a>
+        </li>
+    </ul>
+    <span class="p-more-information">
+        <a class="moresee-link">Get my resume</a>
+        </span>
+        <p class="planguage">Languages</p>
```

Comment:
- [ ] Kindly ensure that your list of technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li tag which will make your website more semantic which in turn will increase your website accessibility 💯